certified-connectors/CandidateZip/readme.md

```diff
@@ -0,0 +1,32 @@
+## CandidateZip Connector
+	CandidateZip is a Global Leader of Simplified Resume/Job Data Extraction into ATS/CRM for Recruiters.
+Typically, we help recruiters who need to extract resumes or jobs into their ATS/CRM quickly. They can easily set up this automated extraction process in three easy steps. We help them save time on data entry so that they can do what they love to (close more jobs).  
+
+
+## Pre-requisites
+You will need the following to proceed:
+* Active Microsoft Flow login
```

Comment:
Please change to "Active Microsoft Power Automate, Power Apps, or Logic Apps account"